src/reference/sectionC_interpretation.inc

```diff
@@ -383,9 +383,9 @@ Interpretation of ``map_variables`` elements
 
 #.  CellML models MUST only contain :code:`map_variables` elements where the interface of variable *A* and the interface of variable *B* are applicable interfaces.
 
-#.  The :code:`variable` elements in a CellML model SHALL be treated as belonging to a single "connected variable set".
-    Each set of connected variables is the set of all :code:`variable` elements for which the corresponding nodes in the variable equivalence network form a connected subgraph.
-    Each set of connected variables represents one variable in the underlying mathematical model.
+#.  The :code:`variable` elements in a CellML model SHALL be treated as belonging to a single "equivalent variable set".
+    Each set of equivalent variables is the set of all :code:`variable` elements for which the corresponding nodes in the variable equivalence network form a equivalent subgraph.
```

Comment:
I think that final connected is actually talking about the connected nodes in the network rather than the connected/equivalent issue :)